### PR TITLE
4.8.13 dev

### PR DESCRIPTION
**Pull request by ravisaurav-tarento**

### CODE REVIEW

File: src/protectedApi_v8/workflow-handler.ts [hunk 1]
```diff
@@ -298,3 +298,38 @@ workflowHandlerApi.post('/userWFApplicationFieldsSearch', async (req, res) => {
         )
     }
 })
+
+workflowHandlerApi.post('/v2/userWFApplicationFieldsSearch', async (req, res) => {
```
Comment (karthik-tarento) [inline src/protectedApi_v8/workflow-handler.ts]: this is older style of implementation we followed... we intercepted all the requests in ui-proxy itself and then called the backend service API from ui-proxy... 
this approach simply adds more load into ui-proxy -- so do use the proxies path for this... 

changes required...
proxies_ts file --> add handler for --> workflowhandler/v2 -- and simply forward the requests to kong... 
whitelist api ---> api path would be --> 
/proxies/v8/workflowhandler/v2/userWFApplicationFieldsSearch

**Review by karthik-tarento — CHANGES_REQUESTED** [review]

File: src/utils/whitelistApis.ts [hunk 2]
```diff
@@ -3013,6 +3013,13 @@ export const API_LIST = {
                 ROLE.SPV_ADMIN,
             ],
         },
+        '/proxies/v8/workflow/v2/userWFApplicationFieldsSearch': {
```
Comment (karthik-tarento) [inline src/utils/whitelistApis.ts:3016]: KONG API points to workflowhanlder service prefix... which means you API should be 
/proxies/v8/workflowhandler/v2/userWFApplicationFieldsSearch

this requires new handler in proxies_v8.ts file too

**Review by karthik-tarento — CHANGES_REQUESTED** [review]

File: src/protectedApi_v8/workflow-handler.ts (hunk 1 above)
Comment (ravisaurav-tarento) [reply src/protectedApi_v8/workflow-handler.ts]: Done

File: src/utils/whitelistApis.ts (hunk 2 above)
Comment (ravisaurav-tarento) [reply src/utils/whitelistApis.ts:3016]: Done

**Pull request merged**